modules/dnn/test/test_torch_importer.cpp

```diff
@@ -351,6 +351,14 @@ TEST_P(Test_Torch_nets, ENet_accuracy)
 #if defined(INF_ENGINE_RELEASE) && INF_ENGINE_VER_MAJOR_EQ(2020010000)
     if (backend == DNN_BACKEND_INFERENCE_ENGINE_NN_BUILDER_2019)
         applyTestTag(CV_TEST_TAG_DNN_SKIP_IE_NN_BUILDER, CV_TEST_TAG_DNN_SKIP_IE_VERSION);
+
+    if (backend == DNN_BACKEND_INFERENCE_ENGINE_NGRAPH && target != DNN_TARGET_CPU)
```

Comment:
Perhaps this should be placed outside of this `#if` (see original location below)

Reply:
Done, thanks!